system-variables.md

```diff
@@ -663,7 +663,7 @@ Query OK, 0 rows affected (0.09 sec)
 ### `tidb_enforce_mpp` <span class="version-mark">New in v5.1</span>
 
 - Scope: SESSION
-- Default value: `OFF`
+- Default value: `OFF`. If you need to change this default value, configure the [`performance.enforce-mpp`](/tidb-configuration-file.md#enforce-mpp) parameter.
```

Comment:
I hope it can be emphasized here that this is a configuration item
```suggestion
- Default value: `OFF`. If you need to change this default value, please modify the [`performance.enforce-mpp`](/tidb-configuration-file.md#enforce-mpp) configuration item
```